pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -151,6 +107,10 @@ func init() {
 
 // Helper Functions
 
+// TODO - Here we're comparing AccountStateStatus to a list of AccountConditionType
```

Comment:
Added this to avoid delaying merging this PR even longer - please shout if you disagree